client/parser/ast.go

```diff
@@ -892,14 +890,6 @@ func (node ASTNode) Updates() ([]model.ScoreUpdate, error) {
 			},
 			voiceEventUpdates...,
 		), nil
-
-	case VoiceNumberNode:
```

Comment:
I have to check, but this also doesn't seem necessary. It's handled in the above `case VoiceNode` and never generated separately.

Reply:
Seems reasonable, since I can't think of a case where we would only want to generate the AST for a VoiceNumberNode by itself :+1: